code/drasil-example/Drasil/GamePhysics/Body.hs

```diff
@@ -434,7 +437,10 @@ goal_statements_list' = map (foldlSent) [goal_statements_G_linear,
   goal_statements_G_angular, goal_statements_G_detectCollision, 
   goal_statements_G_collision]
 
-goal_statements_list = enumSimple 1 (getAcc goalStmt) goal_statements_list'
+goal_label :: Label
+goal_label = mkLabelRALst "goalGM" "goalGM"
```

Comment:
It seems that `goal_label` (and `solution_label` and ...) are duplicated for each example. They should be pulled out into drasil-docLang.